services/venice-controller/src/main/java/com/linkedin/venice/controller/kafka/TopicCleanupService.java

```diff
@@ -168,7 +175,8 @@ void cleanupVeniceTopics() {
           cleanupReplicaStatusesFromMetaSystemStore(topic);
         } catch (Exception e) {
           LOGGER.error(
-              "Received exception while trying to clean up replica statuses from meta system store for topic: {}, but topic deletion will continue",
```

Comment:
Using interpolation `{}` for exception will prevent stack trace from being logged. 

If we want stack trace then the exception should be the last parameter (without corresponding `{}`) in the log statement.